From e20877777ecc2cfa318a7ce330a6ccc523ba713f Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Tue, 5 Feb 2013 13:51:23 -0800
Subject: [PATCH] cpufreq: zynq: Select PM_OPP

Zynq's cpufreq driver relies on the OPP library. Automatically
add it to the kernel when the cpufreq driver is added to ensure a
working setup.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
---
 drivers/cpufreq/Kconfig.arm |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/cpufreq/Kconfig.arm b/drivers/cpufreq/Kconfig.arm
index 8de096f..42afdb8 100644
--- a/drivers/cpufreq/Kconfig.arm
+++ b/drivers/cpufreq/Kconfig.arm
@@ -82,5 +82,6 @@ config ARM_ZYNQ_CPUFREQ
 	depends on ARCH_ZYNQ
 	default n
 	select CPU_FREQ_TABLE
+	select PM_OPP
 	help
 	  This adds the CPUFreq driver for Xilinx Zynq SoCs.
-- 
1.7.1

